.github/workflows/emscripten.yml

```diff
@@ -0,0 +1,64 @@
+# To enable this workflow on a fork, comment out:
+#
+# if: github.repository == 'numpy/numpy'
+name: Test Emscripten/Pyodide build
+
+on:
+  pull_request:
+    branches:
+      - main
+      - maintenance/**
+
+jobs:
+  build-wasm-emscripten:
+    runs-on: ubuntu-latest
+    if: github.repository == 'numpy/numpy'
+    env:
+      PYODIDE_VERSION: 0.22.0a3
+      PYTHON_VERSION: 3.10.2
+      NODE_VERSION: 18
+      EMSCRIPTEN_VERSION: 3.1.24
```

Comment:
Maybe add a comment that these variables can only be changed together and link to 
https://github.com/pyodide/pyodide/blob/main/Makefile.envs#L2 where they are defined for a given Pyodide version.

Reply:
Right. Except for `NODE_VERSION` which is more flexible. As a later refinement, it would be good to fill `PYTHON_VERSION` and `EMSCRIPTEN_VERSION` from the repodata.json file. Maybe we could make a github action that does the setup for a specific Pyodide version.